README.md

```diff
@@ -334,3 +334,73 @@ In this section, we'll add the necessary configuration to work with GitHub Actio
     1. Under "Settings", set the Package Status to "public".
 
 Now you are ready to use the provider, cut releases, and have some well-deserved :ice_cream:!
+
+
+## Building the Provider Locally
+
+There are 2 ways the provider can be built locally:
+
+`make provider` will use the current operating system and architecture to create a binary that can be used on your PATH.
+
+To build the provider for another set of operating systems / architectures, the project uses [goreleaser](https://goreleaser.com/).
+Goreleaser allows a user to build a matrix of binaries. It is a CLI tool and can be used as follows:
+
+```
+archives:
```

Comment:
Please add some explanation as to how his "archives" code block is used.
Do you just toss that into your local make file?
And then you can run the `goreleaser` command as given?